packages/skeleton-svelte/src/components/Toast/types.ts

```diff
@@ -1,111 +1,61 @@
-export interface Toast {
-	/** The unique toast ID. */
-	id?: string;
-	/** The unique toast title text. */
-	title?: string;
-	/** The unique toast description text. */
-	description?: string;
-	/**
-	 * Define the toast type.
-	 * @default info
-	 */
-	type?: 'info' | 'error' | 'success';
-	/** The duration of the toast. Default varies by type. */
-	duration?: number;
-}
-
-// Context ---
-
-// https://zagjs.com/components/react/toast#programmatic-control
-/** Provides access to the Toast Context API. */
 export interface ToastContext {
-	/** Used to create display a new Toast instance. */
-	create: (toast: Toast) => void;
+	toaster: toast.Store;
 }
 
-// Component ---
-
-export interface ToastProviderProps {
-	/**
-	 * Offset from the viewport edge.
-	 * @default bottom-end
-	 */
-	placement?: 'top-start' | 'top-end' | 'bottom-start' | 'bottom-end';
-	/**
-	 * Offset from the viewport edge.
-	 * @default 16px
-	 */
-	offset?: string;
-	/**
-	 * The dismiss button label text.
-	 * @default &times;
-	 */
-	dismissLabel?: string;
-
-	// Group ---
+export interface ToastProviderProps extends toast.StoreProps {
+	// Toast ---
 	/** Provide base classes for the root element. */
 	base?: string;
-	/** Provide gap classes for the root element. */
-	gap?: string;
-	/** Provide z-index classes for the root element. */
-	zIndex?: string;
-	/** Provide arbitrary classes for the root element. */
-	classes?: string;
-
-	// Toast ---
-	/** Provide base classes for the toast cards. */
-	toastBase?: string;
-	/** Provide padding classes for the toast cards. */
-	toastPadding?: string;
-	/** Provide gap classes for the toast cards. */
-	toastGap?: string;
-	/** Provide shadow classes for the toast cards. */
-	toastShadow?: string;
+	/** Provide width classes for the root element. */
+	width?: string;
+	/** Provide padding classes for the root element. */
+	padding?: string;
+	/** Provide border radius classes for the root element. */
+	rounded?: string;
 	/** Provide arbitrary classes for the toast cards. */
-	toastClasses?: string;
+	classes?: string;
 
 	// Message ---
-	/** Provide base classes for the message region. */
+	/** Provide base classes for the message. */
 	messageBase?: string;
-	/** Provide classes for the message title text. */
-	messageTitle?: string;
-	/** Provide classes for the message description text. */
-	messageDescription?: string;
-	/** Provide arbitrary classes for the message region. */
+	/** Provide classes for the message. */
 	messageClasses?: string;
 
+	// Title ---
+	/** Provide base classes for the title. */
+	titleBase?: string;
+	/** Provide classes for the title. */
+	titleClasses?: string;
+
+	// Description ---
+	/** Provide base classes for the description. */
+	descriptionBase?: string;
+	/** Provide classes for the description. */
+	descriptionClasses?: string;
+
 	// Dismiss Button ---
 	/** Provide base classes for the dismiss button. */
 	buttonDismissBase?: string;
```

Comment:
We use `btn` for some components - so maybe we try to match here? It's a bit pedantic I suppose.